CMakeLists.txt

```diff
@@ -53,20 +53,30 @@ set(RAJA_VERSION_PATCHLEVEL 0)
 set(CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/cmake/thirdparty" ${CMAKE_MODULE_PATH})
 
 # Build options
-option(RAJA_ENABLE_OPENMP "Build OpenMP support" On)
-option(RAJA_ENABLE_TARGET_OPENMP "Build OpenMP on target device support" Off)
-option(RAJA_ENABLE_CUDA "Build CUDA support" Off)
-option(RAJA_ENABLE_CLANG_CUDA "Use Clang's native CUDA support" Off)
-set(RAJA_CUDA_ARCH "sm_35" CACHE STRING "Compute architecture to pass to CUDA builds")
-option(RAJA_ENABLE_CUB "Use cub for scans using CUDA" On)
-option(RAJA_ENABLE_TESTS "Build tests" On)
-option(RAJA_ENABLE_EXAMPLES "Build simple examples" On)
-option(RAJA_ENABLE_NESTED "Enable nested loop support" Off)
-option(RAJA_ENABLE_WARNINGS "Enable warnings as errors for CI" Off)
-option(RAJA_ENABLE_DOCUMENTATION "Build RAJA documentation" Off)
+set(ENABLE_OPENMP On CACHE Bool "Build OpenMP support")
```

Comment:
We don't need copy headers - I can delete it.

The reason they have changed to `set` is because BLT already defines them as `option`. The `set` here just alters the default value. Leaving them as `option` makes things all wonky because of how the CMake cache works.